soifunc/deband.py

```diff
@@ -21,30 +18,47 @@ def RetinexDeband(
     "medium" `threshold` in f3kdb is 48. I think that's a bit strong.
     16 might be a more sane starting point. Increase as needed.
 
-    `mask_threshold` determines how sensitive the mask is, lower values
-    should preserve more detail. It does not need to be manually scaled for bit-depth,
+    `mask_threshold` determines how sensitive the mask is.
+    Lower values should preserve more detail.
+    It does not need to be manually scaled for bit-depth,
     this function will do that automatically.
 
     This function does not add grain on its own. Use another function like
-    `kagefunc.adaptive_grain` to do that.
+    `vsdeband.sized_grain` to do that.
     """
+    assert clip.format
+    assert check_variable(clip, retinex_deband)
+
     if (
         clip.format.color_family != vs.YUV
         or clip.format.sample_type != vs.INTEGER
         or clip.format.bits_per_sample > 16
     ):
-        raise value_error("currently only supports 8-16 bit integer YUV input")
+        raise InvalidVideoFormatError(
```

Comment:
I'm not sure how many usages of `value_error` and `type_error` are left around, but if we're moving to iew's error types we can remove basically the entire `internal.py` class

Reply:
All of them were replaced. I can remove it in this PR if you'd like?

Reply:
Thanks, please do that. The intention was never for them to be used outside of this package, so they're dead code now. :+1: